stdpopsim/genomes.py

```diff
@@ -119,6 +172,13 @@ class Contig:
         <https://msprime.readthedocs.io/en/stable/api.html#msprime.RecombinationMap>`_
         for more details.
     :vartype recombination_map: msprime.simulations.RecombinationMap
+    :ivar genomic_element_types: a list of stdpopsim.GenomicElementType
+        objects. By default, the only element is a single genomic element type
+        that spans the entire contiguous region.
+    :vartype genomic_element_types: list
+    :ivar mutation_types: a list of stdpopsim.ext.MutationType object. By
+        default, the only mutation type is neutral. See
+        stdpopsim.MutationType for more details.
```

Comment:
Probably good to note here that these are only relevant when using the SLiM engine.